go/vt/tabletserver/sqlquery_test.go

```diff
@@ -410,7 +410,8 @@ func TestHandleExecUnknownError(t *testing.T) {
 		BindVariables: nil,
 	}
 	var err error
-	defer handleExecError(&query, &err, logStats)
+	sq := &SqlQuery{}
```

Comment:
There are two other places, but I'll leave them alone because they modify the state. So, they could poison the other tests.